packages/slider/src/presenters/Input.stylesheet.js

```diff
@@ -0,0 +1,172 @@
+const thumbPseudoElements = {
+  microsoft: "::-ms-thumb",
+  mozilla: "::-moz-range-thumb",
+  webkit: "::-webkit-slider-thumb"
+};
+
+const trackPseudoElements = {
+  microsoft: "::-ms-track",
+  mozilla: "::-moz-range-track",
+  webkit: "::-webkit-slider-runnable-track"
+};
+
+const trackProgressPseudoElements = {
+  // WebKit does not have a pseudo element to target this styling, so we achieve it with fancy styling on track
+  // ::-webkit-slider-runnable-track
+  microsoft: "::-ms-fill-lower",
+  mozilla: "::-moz-range-progress"
+};
+
+function stylesheet(themeData, trackValueRatio) {
+  function browserSpecificPseudoElementRules(
+    browserPseudoElements,
+    rules,
+    singleBrowserInclusions
+  ) {
+    const styles = {};
+
+    Object.entries(browserPseudoElements).forEach(browserPseudoElement => {
+      let pseudoElementRules = rules;
+
+      const browser = browserPseudoElement[0];
+      const pseudoElement = browserPseudoElement[1];
+
+      if (singleBrowserInclusions && singleBrowserInclusions[browser]) {
+        pseudoElementRules = Object.assign(
+          {},
+          rules,
+          singleBrowserInclusions[browser]
+        );
+      }
+
+      styles[`&${pseudoElement}`] = pseudoElementRules;
+    });
+
+    return styles;
+  }
+
+  const baseRules = {
+    WebkitAppearance: "none" /* Hides platform-native styling */,
+    width: "100%" /* Specific width is required for Firefox. */,
+    backgroundColor: "transparent" /* Otherwise white in Chrome */,
+    boxSizing: "content-box",
+    height: "20px",
+    margin: 0,
+    position: "relative",
+    outline: "none",
+
+    "&::-ms-tooltip": {
+      display: "none"
+    },
+
+    "&::-moz-focus-outer": {
+      border: 0
+    }
+  };
+
+  const thumbRules = browserSpecificPseudoElementRules(
+    thumbPseudoElements,
+    Object.assign(
+      {},
+      {
+        boxSizing: "content-box",
+        height: themeData["slider.thumb.width"],
+        width: themeData["slider.thumb.width"],
+        backgroundColor: themeData["slider.thumb.backgroundColor"],
+        border: 0,
+        borderRadius: "50%",
+        boxShadow: `0 0 0 ${themeData["slider.halo.width"]} ${
+          themeData["slider.halo.color"]
+        }`,
+        cursor: "pointer",
+        position: "relative",
+        top: "50%",
+        WebkitAppearance: "none" /* Hides platform-native styling */,
+
+        "&:hover": {
```

Comment:
The preference is to use the `hasHover`, `hasFocus`, and `isPressed` props where possible instead of using `css`. You can get these from using the `Behaviors` component. You can see examples with `input`, `radio-button`, and `checkbox`.